app/brave_generated_resources.grd

```diff
@@ -886,6 +886,15 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
       <message name="IDS_BRAVE_IPFS_INFOBAR_LINK" desc="Link redirects to the privacy description article">
         Learn how this affects my privacy
       </message>
+      <message name="IDS_BRAVE_IPFS_FALLBACK_INFOBAR_TEXT" desc="Text that propose user to load an original urls instead of resolved one, due to page loading error">
```

Comment:
@vadimstruts — just fixing a couple of typos here. Should be:

<message name="IDS_BRAVE_IPFS_FALLBACK_INFOBAR_TEXT" desc="Prompt, asking the user whether they'd like to load the original URL rather than the resolved one when the page load fails">

Reply:
`Prompt, asking the user whether they'd like to load the original URL rather than the resolved one when the page load fails`

Reply:
fixed